src/fetchRequests.js

```diff
@@ -21,6 +21,27 @@ const fetchRequests = {
       .then(response => response.json())
       .then(data => data.trips)
       .catch(error => console.log(error))
+  },
+  postNewTrip(trip) {
```

Comment:
Could deconstruct here - not absolutely necessary, depends on the developer... Just a cool thing to know about ES6

https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6